e2e/manifests/values/daemonset-lvmd-storage-capacity.yaml

```diff
@@ -0,0 +1,109 @@
+image:
+  repository: topolvm
+  tag: dev
+  pullPolicy: Never
+
+controller:
+  replicaCount: 1
+  storageCapacityTracking:
+    enabled: true
+  securityContext:
+    enabled: false
+  # sanity test requires that the controller mounts this hostPath to communicate with it
+  volumes:
+    - name: socket-dir
+      hostPath:
+        path: /var/lib/kubelet/plugins/topolvm.cybozu.com/controller
+        type: DirectoryOrCreate
+
+scheduler:
+  type: deployment
+  deployment:
+    replicaCount: 0
```

Comment:
Would you add a `enable` flag instead of decreasing replicaCount?

Reply:
Thanks, I added `scheduler.enabled` option.
https://github.com/topolvm/topolvm/pull/325/commits/5817656e84b893a47de69c23e45a8d38f33594f2